baseplate/clients/redis_cluster.py

```diff
@@ -104,6 +108,31 @@
 #  of key value [key value ...]
 MULTI_KEY_BATCH_WRITE_COMMANDS = frozenset(["MSET", "MSETNX"])
 
+PROM_NAMESPACE = "redis_cluster"
```

Comment:
same here, metrics are `redis_cluster_client_` and labels are just `redis_cluster_`